### DISCUSSION TITLE
prodcessor_add_fields是否可以增加环境变量字段？

### DISCUSSION THREAD

**Discussion by q121102302** [alibaba/ilogtail#752] [Help] [answered]
<img width="244" alt="image" src="https://user-images.githubusercontent.com/101638329/228835240-0b8525c7-60b9-462a-9ff6-ab7810d3189f.png">
prodcessor_add_fields中增加环境变量没有生效。是不支持还是我的写法有问题？
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/101638329/228835649-22f3607a-71fa-42fa-bb0a-925810efccb3.png">

**Comment by messixukejia** [comment, marked as answer]
关注下 https://github.com/alibaba/ilogtail/discussions/724
https://github.com/alibaba/ilogtail/pull/746
